apptail/docker/listener.go

```diff
@@ -65,3 +67,12 @@ func (l *dockerListener) Listen() {
 		l.mux.Unlock()
 	}
 }
+
+func (l *dockerListener) TrackerCleanUp(tracker storage.Tracker) {
+	all_containers := docker_events.GetLiveDockerContainers(RETRY)
+	if len(all_containers) > 0 {
```

Comment:
So, if there are no live containers, we don't do cleanup? (I guess we assume that's the case of failure to get containers?)  Just seemed a little odd.